plugins/modules/junos_acls.py

```diff
@@ -98,7 +98,12 @@
                   prefix_list:
                     description:
                     - IP source prefix list to use for the filter
-                    type: str
+                    type: list
+                    elements: dict
+                    suboptions:
```

Comment:
Any specific reason why the elements of the list are dict, since it consists of only one option it can be an `str`?